site/styles/00-tokens-base.css

```
@@ -0,0 +1,60 @@
+  --line:#aeb8bd;
+  --line-dark:#5e6c77;
+  --blue:#2457ff;
+  --blue-deep:#1238b6;
+  --blue-pale:#dce5ff;
+  --signal:#ff5a36;
+  --signal-text:#a82c12;
+  --signal-pale:#ffdcd3;
+  --lime:#ccff3d;
+  --white:#fff;
+  --font-display:"Archivo Black","Arial Black",sans-serif;
+  --font-body:"Atkinson Hyperlegible",Arial,sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)